streaming/base/storage/download.py

```diff
@@ -186,7 +186,7 @@ def download_from_local(remote: str, local: str) -> None:
     os.rename(local_tmp, local)
 
 
-def download(remote: Optional[str], local: str, timeout: float):
+def download_file(remote: Optional[str], local: str, timeout: float):
```

Comment:
is it unclear that these are files? also doesn't match `download_or_wait` etc

Reply:
The download method is actually downloading the file and since we are now importing the method `download()` from the filename `download.py` in `__init__.py`,  python was getting confused where it is a method or a file. Hence, I changed the method name.